NEWS.md

```diff
@@ -1,3 +1,41 @@
+### v8.11.0 (2022-05-11)
+
+* Log forwarding from APM agent is now possible.
+
+    * Added application log aggregator to collect logs with adaptive sampling and appropriate max samples stored.
+
+    * Added `application_logging` configuration and env vars with appropriate defaults.
+
+    * Changed default value of `application_logging.enabled` to be true.
+
+    * Set `application_logging.forwarding.enabled` to false when HSM is set.
+
+    * Enabled log forwarding by default in the example config.
+
+    * Added sent, seen and dropped metrics that collected on every harvest cycle around log lines.
+
+    * Added supportability metrics for some popular logging frameworks.
+    
+    * Added supportability metrics to record if the logging features are enabled.
+
+    * Added a storage mechanisms to transactions to keep logs until transaction ends.
+
+    * Updated log aggregator to check transaction context before adding.
```

Comment:
probably irrelevant, details of the churn of this feature

Reply:
agreed. let's make sure the notes reflect the final product with sufficient detail.